meetings/2019-04-09.md

```diff
@@ -0,0 +1,14 @@
+# TSC Meeting April 9, 2019 @12PM MST
+
+## [Hangouts Link](https://hangouts.google.com/hangouts/_/doi.gov/jlaura)
+
+## Agenda
+- Organizing a planetary science software group.
```

Comment:
Was there nothing else...?

Reply:
I'm waiting for @jlaura to post last meeting's notes with stuff to continue discussion of. Also, people can add other agenda items. This is just something that we talked about here at LPSC.

Reply:
My slacker self got the PR opened.